test/Fail/Issue1944-UnappliedOverloadedProjectionNotRecord.err

```diff
@@ -1,4 +1,7 @@
 Issue1944-UnappliedOverloadedProjectionNotRecord.agda:15,8-9
-Cannot resolve overloaded projection f because principal argument
-is not of record type
-when checking that the expression f has type A → A
+Cannot resolve overloaded projection 'f' because
+principal argument has type 'A₁'  while it should be of record type
+Signatures in the scope: 
+- f : {@0 B : Set} → R B → B
+- f : {@0 B : Set} → S B → B
+when checking that the expression f has type A₁ → A₁
```

Comment:
`A₁ → A₁` is looks worse at first in comparison to `A → A`, but when looking at the testcase, there is an `A` in scope which is unrelated, so `A₁` could fit some taste better as it is distinct to `A`, even though the user didn't write `A₁` anywhere.
In the olden days, we printed such as `.A → .A` where the preceding `.` was used for "not in scope", but this was also confusing.  Best would be colored printing, then we could have a different color for out-of-scope identifiers.

Reply:
We should also distinguish clashing identifiers by a suffix, where possible, of course.